### PR TITLE
Docs and Unit Tests

### PR DESCRIPTION
**Pull request by Paxanator**
This PR adds bare bones docs and unit tests to get us going. Next PR will be travis integration. This is progress towards both #5 and #8 but doesn't achieve either fully

### CODE REVIEW

**Comment by Paxanator** [comment]
This does address #18

File: fRanz/tests/testthat/helper-kafka-vars.R [hunk 1]
```diff
@@ -0,0 +1,2 @@
+KAFKA_HOST <- "localhost"
```
Comment (mjermann) [inline fRanz/tests/testthat/helper-kafka-vars.R:1]: don't you need a `docker-compose` call to ensure the kafka service is running at `localhost`?

**Review by mjermann — COMMENTED** [review]
This looks great - my only question is regarding the `testthat` logistics. If you can address that we should be good to merge!

File: fRanz/tests/testthat/helper-kafka-vars.R (hunk 1 above)
Comment (Paxanator) [reply fRanz/tests/testthat/helper-kafka-vars.R:1]: I'm checking these in now, and then will be adding a .travis.yml to do docker-compose and then test!

Comment (Paxanator) [reply fRanz/tests/testthat/helper-kafka-vars.R:1]: I have verified they work locally

**Review by mjermann — APPROVED** [review]
LGTM!

**Pull request merged**